### ISSUE TITLE
from simple_knn._C import distCUDA2

### ISSUE THREAD

**Issue by BruceYounggit** [hustvl/4DGaussians#195]
I've installed "simple-knn" using the command "pip install -e submodules/simple-knn". However, when I run the script gaussian_model.py, I encounter an error stating that "from simple_knn._C import distCUDA2" is incorrect. Upon checking the simple_knn library in the Remote Libraries, I found only one file named '.gitkeep' instead of "._C". Does anyone know what might have caused this discrepancy?

**Comment by BladeRunner132** [comment]
I have the same problem as yours. Wish someone could give us a hand.

**Comment by LearnToStick** [comment]
I have a same problem.Have you solved it？

**Comment by guanjunwu** [comment]
I didnt encount this problem. Do you check your cuda version which should corrsponds to torch's version？

**Comment by BruceYounggit** [comment]
Thanks. I solved it by using another function to replace distCUDA2.


def distCUDA2(points):
    points_np = points.detach().cpu().float().numpy()
    dists, inds = KDTree(points_np).query(points_np, k=4)
    meanDists = (dists[:, 1:] ** 2).mean(1)
    return torch.tensor(meanDists, dtype=points.dtype, device=points.device)


> I didnt encount this problem. Do you check your cuda version which should corrsponds to torch's version？
> I have a same problem.Have you solved it？
> I have the same problem as yours. Wish someone could give us a hand.

**Comment by utkarshpawade** [comment]
I was getting the same error when i was trying to run it in colab. So I fixed it by adding this in gaussian_model.py
import sys
sys.path.append("/content/4DGaussians/submodules/simple-knn")
from simple_knn._C import distCUDA2
